priv/repo/migrations/20190501221754_create_avatar_listings.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Ret.Repo.Migrations.CreateAvatarListings do
+  use Ecto.Migration
+
+  def change do
+    Ret.AvatarListing.State.create_type()
+
+    create table(:avatar_listings, prefix: "ret0", primary_key: false) do
+      add(:avatar_listing_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true)
+      add(:avatar_listing_sid, :string)
```

Comment:
`null: false`?

Reply:
Hmm copped from scene_listings, but that does seem right